modules/ROOT/pages/rtc/rtc-supported-functionality.adoc

```diff
@@ -56,23 +56,23 @@ When planning to integrate Real-time Collaboration into your application or webs
 
 == Options that are not available when RTC is enabled
 
-* link:{baseurl}/configure/content-filtering/#allow_unsafe_link_target[`allow_unsafe_link_target`]
-* link:{baseurl}/configure/advanced-editing-behavior/#br_in_pre[`br_in_pre`]
-* link:{baseurl}/configure/content-filtering/#custom_elements[`custom_elements`]
-* link:{baseurl}/configure/advanced-editing-behavior/#end_container_on_empty_block[`end_container_on_empty_block`]
-* link:{baseurl}/configure/content-filtering/#extended_valid_elements[`extended_valid_elements`]
-* link:{baseurl}/configure/content-filtering/#forced_root_block[`forced_root_block`]
-* link:{baseurl}/configure/content-filtering/#forced_root_block_attrs[`forced_root_block_attrs`]
-* link:{baseurl}/configure/content-formatting/#format_empty_lines[`format_empty_lines`]
-* link:{baseurl}/configure/spelling/#gecko_spellcheck[`gecko_spellcheck`]
-* link:{baseurl}/plugins/opensource/image/#image_caption[`image_caption`]
-* link:{baseurl}/configure/file-image-upload/#images_dataimg_filter[`images_dataimg_filter`]
-* link:{baseurl}/configure/content-formatting/#indent_use_margin[`indent_use_margin`]
-* link:{baseurl}/configure/content-appearance/#inline_boundaries[`inline_boundaries`]
-* link:{baseurl}/configure/content-appearance/#inline_boundaries_selector[`inline_boundaries_selector`]
-* link:{baseurl}/configure/content-filtering/#keep_styles[`keep_styles`]
-* link:{baseurl}/configure/editor-appearance/#placeholder[`placeholder`]
-* link:{baseurl}/configure/content-filtering/#protect[`protect`]
-* link:{baseurl}/plugins/opensource/link/#rel_list[`rel_list`]
-* link:{baseurl}/configure/content-filtering/#valid_children[`valid_children`]
-* link:{baseurl}/configure/content-filtering/#valid_elements[`valid_elements`]
+* xref:configure/content-filtering.adoc#allow_unsafe_link_target[`allow_unsafe_link_target`]
+* xref:configure/advanced-editing-behavior.adoc#br_in_pre[`br_in_pre`]
+* xref:configure/content-filtering.adoc#custom_elements[`custom_elements`]
+* xref:configure/advanced-editing-behavior.adoc#end_container_on_empty_block[`end_container_on_empty_block`]
+* xref:configure/content-filtering.adoc#extended_valid_elements[`extended_valid_elements`]
+* xref:configure/content-filtering.adoc#forced_root_block[`forced_root_block`]
+* xref:configure/content-filtering.adoc#forced_root_block_attrs[`forced_root_block_attrs`]
+* xref:configure/content-formatting.adoc#format_empty_lines[`format_empty_lines`]
+* xref:configure/spelling.adoc#gecko_spellcheck[`gecko_spellcheck`]
+* xref:plugins/opensource/image.adoc#image_caption[`image_caption`]
+* xref:configure/file-image-upload.adoc#images_dataimg_filter[`images_dataimg_filter`]
+* xref:configure/content-formatting.adoc#indent_use_margin[`indent_use_margin`]
```

Comment:
This xref doesn't resolve as the `indent_use_margin` id doesn't exist. It looks like the wrong id was used in the `indentation.adoc` file as it's used `[[images_dataimg_filter]]` instead of `[[indent_use_margin]]`.

- [ ] Fix broken xref

Reply:
Realised I could leave a comment on the incorrect id below instead, so ignore this comment.